contract/tests/test_contract.py

```diff
@@ -136,11 +136,15 @@ def _add_template_line(self, overrides=None):
 
     def test_add_modifications(self):
         partner2 = self.partner.copy()
-        subtype = self.env.ref('contract.mail_message_subtype_contract_modification')
         self.contract.message_subscribe(
             partner_ids=partner2.ids,
-            subtype_ids=subtype.ids
+            subtype_ids=self.env.ref('mail.mt_comment').ids
```

Comment:
Why changing this?

Reply:
Because try to add new followers without modification subtype to test after.

Reply:
But I don't see that you subscribe to this type after, so you are relying on default values.

Reply:
Please do the test in a more resilient way.